### PR TITLE
[Smotri] Remove extractor

### PR DESCRIPTION
**Pull request by mashed-potatoes**
### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

On July 30, 2018, the _Smotri dot com_ radically changed its direction of activity (it may have been bought out by persons affiliated with Bongacams, a network of erotic video chat sites).
At the moment, the site is a mirror of the Bongacams. Hence there's no point in the extractor, so it should be removed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -371,7 +371,7 @@ Alternatively, refer to the [developer instructions](#developer-instructions) fo
                                      out, youtube-dl will ask interactively.
     -2, --twofactor TWOFACTOR        Two-factor authentication code
     -n, --netrc                      Use .netrc authentication data
-    --video-password PASSWORD        Video password (vimeo, smotri, youku)
```
Comment (dstftw) [inline README.md:374]: This file is autogenerated. Do not edit.

File: docs/supportedsites.md [hunk 2]
```diff
@@ -817,10 +817,6 @@
  - **Slideshare**
  - **SlidesLive**
  - **Slutload**
- - **smotri**: Smotri.com
- - **smotri:broadcast**: Smotri.com broadcasts
- - **smotri:community**: Smotri.com community videos
- - **smotri:user**: Smotri.com user videos
```
Comment (dstftw) [inline docs/supportedsites.md:823]: This file is autogenerated. Do not edit.

**Review by dstftw — CHANGES_REQUESTED** [review]

**Comment by mashed-potatoes** [comment]
Perhaps I should create a new PR to clean out the commit history.

**Comment by TeslaLyon** [comment]
Please add support Bongacams.com.

https://github.com/ytdl-org/youtube-dl/issues/27440

**Pull request merged**